modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,344 @@
+= MongoDB - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Upgrade Anypoint Connector for MongoDB (MongoDB Connector) to version 6.0.0.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|5.3.x |6.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* Changed operation names, metadata, fields, and values
++
+*Reviewers, what values were changed? I do not see them in the PR. Also, were some operations added besides the Object Listener input source?*
```

Comment:
There were changes in the connection timeout values, which went from being expressed in milliseconds to seconds.

Reply:
No more operations were added, and the Object Listener operation is a replacement for New Object.

Reply:
We should list which fields changed.